YogaKit/YGLayout.m

```diff
@@ -318,6 +312,14 @@ static void YGAttachNodesFromViewHierachy(UIView *const view)
   if (yoga.isLeaf) {
     YGRemoveAllChildren(node);
     YGNodeSetMeasureFunc(node, YGMeasureView);
+
+    YGSize oldSize = yoga.cachedSize;
+    YGSize newSize = YGMeasureView(node, 0, YGMeasureModeUndefined, 0, YGMeasureModeUndefined);
```

Comment:
It's not easy necessarily easy to implement within applications outside of yoga: it requires wrapping a call to `applyLayout` with a function which visits the view hierarchy, finding the leaf nodes, caching the view's size and dirty it only if the size changes. I know iOS fairly well, and I don't yet see a better way to do it. Swizzling `setNeedsLayout` was a good idea, but it didn't work in practice.